Packs/CommonScripts/Scripts/FormatTemplate/test_data/test.json

```diff
@@ -0,0 +1,177 @@
+[
+  {

```

Comment:
Please provide command examples with inputs for both commands and document the outputs. This way it will be much more clearer for the user how to use the commands.

Reply:
@anara123 OK. I've documented them to README.